netstandard/ref/System.Threading.Tasks.cs

```diff
@@ -262,6 +262,7 @@ public enum TaskCreationOptions
     }
     public static partial class TaskExtensions
     {
+        public static System.Runtime.CompilerServices.ConfiguredAsyncEnumerable<T> ConfigureAwait<T>(this System.Collections.Generic.IAsyncEnumerable<T> source, bool continueOnCapturedContext) { throw null; }
```

Comment:
Ditto.
https://github.com/dotnet/corefx/issues/33909

There's also a question again of whether this is actually where we want to expose this method and the associated WithCancellation, vs putting them on some new type.